sqlglot/dialects/duckdb.py

```diff
@@ -116,17 +116,24 @@ def _build_make_timestamp(args: t.List) -> exp.Expression:
 
 def _struct_sql(self: DuckDB.Generator, expression: exp.Struct) -> str:
     args: t.List[str] = []
+
+    # BigQuery allows inline construction such as "STRUCT<a STRING, b INTEGER>('str', 1)" which is
+    # canonicalized to "ROW('str', 1) AS STRUCT(a TEXT, b INT)" in DuckDB
+    is_struct_cast = expression.find_ancestor(exp.Cast)
```

Comment:
What triggered the need to make the change in `_struct_sql`? Why are we converting into `ROW`? Can't we just use the `{ ... }` syntax?